### ISSUE TITLE
Flaky test test_batch_gp_regression.py

### ISSUE THREAD

**Issue by Balandat** [cornellius-gp/gpytorch#162]
This test was labeled as flaky by our CI system. 1/50 runs would produce the following test failure:

```
RuntimeError: Lapack Error syev : 9 off-diagonal elements didn't converge to zero at caffe2/aten/src/TH/generic/THTensorLapack.cpp:388
  File "/usr/local/fbcode/gcc-5-glibc-2.23/lib/python3.6/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/usr/local/fbcode/gcc-5-glibc-2.23/lib/python3.6/unittest/case.py", line 605, in run
    testMethod()
  File "test/examples/test_batch_gp_regression.py", line 61, in test_posterior_latent_gp_and_likelihood_with_optimization
  File "gpytorch/module.py", line 162, in __call__
  File "gpytorch/mlls/exact_marginal_log_likelihood.py", line 36, in forward
  File "gpytorch/lazy/lazy_variable.py", line 469, in inv_quad_log_det
  File "gpytorch/functions/_inv_quad_log_det.py", line 129, in forward
  File "gpytorch/utils/lanczos.py", line 170, in lanczos_tridiag_to_diag
  File "gpytorch/utils/eig.py", line 23, in batch_symeig
```

@jacobrgardner, is there an easy way to make this more robust?

**Comment by jacobrgardner** [comment]
I'll take a look.

**Comment by jacobrgardner** [comment]
I think this is addressed by #164 

If we still have trouble, we can either lock down the seed for that test, or remove optimization and fix the hyperparameters so that only testing is done during that test (since the batch GP part of the test happens after optimization).
